### PR TITLE
Next CI Test: Biomed Data

### PR DESCRIPTION
**Pull request by fppt**
This PR takes the data from [this](https://github.com/graknlabs/biomed) repo and turns it into a CI test.

At this stage it is not configured to run on the CI platform as I am waiting for https://github.com/graknlabs/grakn/pull/2072 to go in before linking things together.

### CODE REVIEW

File: grakn-test/test-biomed/validation/validate.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+runQuery(query, 54)
+
+print ("Running explicit query . . .")
+query = "match  $cancer isa cancer has name \"breast cancer\";" \
+    + "($cancer, $sl); $sl isa stem-loop;" \
+    + "($sl, $m); $m isa mature;" \
+    + "($m, $gene); $gene isa gene;" \
+    + "($gene, $drug); $drug isa drug;" \
+    + "offset 0; limit 1;" \
+    + "aggregate count;"
```
Comment (aelred) [inline grakn-test/test-biomed/validation/validate.py]: You can do this much more neatly in Python with triple quotes:

```
query = """
match  $cancer isa cancer has name "breast cancer";
($cancer, $sl); $sl isa stem-loop
($sl, $m); $m isa mature;
($m, $gene); $gene isa gene;
($gene, $drug); $drug isa drug;
offset 0; limit 1;
aggregate count;
"""
```

File: grakn-test/test-biomed/validation/validate.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            print '%r  %2.2f ms' % \
+                  (method.__name__, (te - ts) * 1000)
+        return result
+    return timed
+
+@timeit
+def runQuery(query, numResults):
+    result = check_output(["graql.sh", "-n", "-k", "biomed", "-e", query]);
+    assert int(result) == numResults
+    return
```
Comment (aelred) [inline grakn-test/test-biomed/validation/validate.py]: This return isn't really necessary!

File: grakn-test/test-biomed/validation/validate.py (hunk 1 above)
Comment (aelred) [reply grakn-test/test-biomed/validation/validate.py]: This also means you don't need to escape the " quotes.

**Comment by fppt** [comment]
@aelred re-review please

**Pull request merged**